src/redux/modules/frontPage.ts

```diff
@@ -31,7 +32,7 @@ export type ReduxState = {
 
 // returns true if filterValue is found (includes()) within any element of termsToSearch
 export const textInDetails = (filterValue: string, termsToSearch: [string, string]) =>
-  any((phrase) => phrase.includes(filterValue), termsToSearch)
+  any((phrase) => phrase.includes(removeDiacritics(filterValue)), termsToSearch)
```

Comment:
Yeah, I misread. Sorry for the trouble.

I moved it to the `filterEpic`, but I couldn't figure out where to test this (I don't have any experience in `redux-observable`. Because in `textInDetails` it already arrives "harmonized" and using diacritics in `routingEpic` should be fine I think.